practica2023/src/main/java/com/endava/practica2023/controller/OrderController.java

```diff
@@ -24,4 +30,24 @@ public OrderController(OrderService orderService) {
     public List<Order> getOrderList(@RequestParam Integer id){
         return orderService.getOrderById(id);
     }
+
+    @Autowired
+    private TicketCategoryRepo ticketCategoryRepo;
+
+    @Autowired
+    private OrderRepo orderRepo;
+    @PostMapping("/POST_orders")
+    public UsefulInfo postOrder(@RequestBody NewOrder newOrder){
+        LocalDateTime localDateTime= LocalDateTime.now();
+        TicketCategory ticketCategory= ticketCategoryRepo.findById(newOrder.getTicketCategoryId()).get();
+        this.user=new User();
+        User user2=userRepo.findById(1).get();
+
+        System.out.println(user2);
+
+        Order order = new Order(localDateTime, newOrder.getNumberOfTickets(), newOrder.getNumberOfTickets()*ticketCategory.getPrice(),
+                user2, ticketCategory);
+        Order order2 = orderRepo.save(order);
+        return orderService.usefulInfo(order2);
```

Comment:
If you want to control how the response looks you should look into creating DTOs. You can make a dedicated converter class that transforms the model object into the DTO you need. UsefulInfo is a very ambiguous name, OrderDTO or OrderResponse is more suggestive.